### DISCUSSION TITLE
Transaction

### DISCUSSION THREAD

**Discussion by Alex-MRX-777** [btcpayserver/btcpayserver#4438] [Technical Support]
When transferring from wallet to wallet using the web interface, I see how much the mining fee will be, but when using the API, I don’t know if it’s possible to add it.

**Comment by bolatovumar** [comment]
@Alex-MRX-777 This should give you the fee rate: https://docs.btcpayserver.org/API/Greenfield/v1/#operation/StoreOnChainWallets_GetOnChainFeeRate

Or are you looking for the total fee paid when transaction was broadcasted?

**Reply by Alex-MRX-777** [reply]
Or are you looking for the total fee paid when transaction was broadcasted? - Yes

**Reply by bolatovumar** [reply]
Yeah, it can be done but it's sort of annoying to do.
